src/strategy-factories/call-tokenuri/extractor.mjs

```diff
@@ -66,7 +70,7 @@ export const callTokenUriFactory = (props) => {
           to: address,
           data,
         },
-        toHex(BLOCK_NUMBER),
+        toHex(Math.max(parseInt(blockNumber), BLOCK_NUMBER)),
```

Comment:
For NFTs minted after `BLOCK_NUMBER` we will have to use a higher block number. Perhaps the one coming from `createdAtBlockNumber`.

Reply:
https://github.com/neume-network/strategies/commit/edc666543c6160a334f702ec68a79b2c3f3dea67